opre_ops/ops_site/migrations/0002_auto_20210901_2327.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.6 on 2021-09-01 23:27
+
+import os
+
+from django.db import migrations
+from csv import DictReader
+from ops_site.models import Agency
+
+def seed_agencies(apps, schema_editor):
+    filename = os.path.join('ops_site', 'migrations', 'seed_data', 'fake_data_agencies.csv')
+    with open(filename, encoding="utf-8-sig") as read_obj:
+        csv_dict_reader = DictReader(read_obj)
+        for row in csv_dict_reader:
+            agency = Agency(name=row.get("Name"), nickname=row.get("Nickname"))
+            agency.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ops_site', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RunPython(seed_agencies)
+    ]
```

Comment:
[Thought, Non-Blocking] All of our migrations will run when we deploy the application code in a new environment. Seeding this fake data with a data migration will work for development, but in production we won't want to seed our database with fake data. 

Options we could consider:

### 1. Don't deal with this now

Keep seeding fake/development data using database migrations. When we get to production, we may need to change or delete these development data migration files and adopt one of the strategies below. 

### 2. Move data to a Django fixture

Relevant docs: https://docs.djangoproject.com/en/3.2/howto/initial-data/

We could move our seed data to JSON or YAML and load using `python manage.py loaddata <fixturename>`. We could add the `loaddata` task to our `Dockerfile` or `docker-compose` setup for local development, and use a settings flag to have it run in our cloud.gov Development/Demo environments, but not in Staging or Production. 

### 3. Add `if` logic to database migrations for seed data

Add custom logic or flags to our database migrations that seed data, so that they only run in certain environments.

--- 

@amymok @carjug Let me know what you think! How have you approached this in other projects? This comment is non-blocking because I would be totally fine with option (1). Looks like option (2) might be more Django-idiomatic over the long run?

Reply:
I prefer option (2).  This way we can keep the migration clear from dev data.  But I agree that we can add a new ticket to move that option.  The only thing I am unsure about is, if we do that later, how does that work to "remove" a migration file?  Will that give us more headaches later when we have other new migrations?  Or is it easier to do that now and remove this migration before we add new one?

Reply:
@amymok Yep, I agree that removing migration files might get slightly messy because databases (local, deployed, etc) might be in different states. So that would suggest making the switch over to using Django fixtures in this PR rather than in a follow-on issue or PR. 

@carjug What do you think?

Reply:
I can do that. There's a bit of manual labor involved because the file structure for Fixtures is not easily exportable from the AirTable but I can definitely do it.

Reply:
@carjug Ooooh, maybe `python manage.py dumpdata` could give you the data in a fixture-friendly way without manual labor? 

https://docs.djangoproject.com/en/3.2/ref/django-admin/#django-admin-dumpdata

Reply:
@carjug Since future issues will follow a similar pattern, maybe a lightweight CSV-to-JSON or CSV-to-YAML script will be helpful to us in the future? Like to get us a quick "export Airtable CSV" > "transform to JSON or YAML" pipeline.

Reply:
@carjug You've addressed this by moving to a fixture pattern, excellent! Thank you!